docs/1_python.md

```diff
@@ -2,7 +2,9 @@
 
 Oras Python will allow you to run traditional push and pull commands for artifacts,
 or generate a custom client. This small user guide will walk you through these various
-steps, and please open an issue if functionality is missing.
+steps, and please [open an issue](https://github.com/oras-project/oras-py/issues) if 
+functionality is missing. More detailed developer examples can also be found on the 
+[Oras Python](https://oras-project.github.io/oras-py/) page hosted alongside the repository.
```

Comment:
Do you think it will be better to combine [Oras Python](https://oras-project.github.io/oras-py/) to the [official website](https://oras.land/)? People might be easier to find ORAS-python from a unified documentation portal.

Reply:
Just noticed you are working on migrating the docs from oras-py to the ORAS website. Let me know if you need assistance.

Reply:
That's the entire point of this PR - the file here 1_python.md is grabbed and deployed to the official site! It should be done after these two are merged.

Reply:
This kills two birds with one stone because the corresponding docs are alongisde the respective code (e.g., as I'm updating the library here I can update them) but then nightly they are grabbed and deployed to oras-www. Here is the page here deployed there: https://oras.land/client_libraries/1_python/ and the PR here is removing the redundant docs from here.